fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -176,6 +176,8 @@
 
     public static final String BLOCK_ENCRYPTION_MODE = "block_encryption_mode";
 
+    public static final String ENABLE_STAR_SCHEMA_RUNTIME_FILTER_POLICY = "enable_star_schema_runtime_filter_policy";
```

Comment:
Under star schemas such as SSB, after `set enable_star_schema_runtime_filter_policy=true`.
Will performance overall be higher than the default RF generation rules?

Reply:
In SSB test set, this policy will benefit q2.1, q2.2, q2.3, q4.1 and q4.3, and don't have any impact on other sqls. In tpch test set, q9, q10, q17 and q18 would run faster after removing the useless runtime filters, no impact on other sqls either.

Reply:
In SSB test set, this policy will benefit q2.1, q2.2, q2.3, q4.1 and q4.3, and don't have any impact on other sqls. In tpch test set, q9, q10, q17 and q18 would run faster after removing the useless runtime filters, no impact on other sqls either.